py/tests/plane_hog.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python3
+
+import pykms
+import time
+import random
+
+card = pykms.Card()
+res = pykms.ResourceManager(card)
+conn = res.reserve_connector()
```

Comment:
Your script presumes that we have lcd and hdmi, and hdmi is the second one.

You could rather have two variables at the start, conn1_name and conn2_name, or something. And have the names there. Then it's explicit and easy to change.